PACKAGES.md

```diff
@@ -0,0 +1,103 @@
+# Packaging Frescobaldi
+
+## Versioning and release best practices
+
+Release versioning should follow [PEP 440](https://peps.python.org/pep-0440/).
```

Comment:
```suggestion
Release versioning should follow [Python versioning recommendations](https://packaging.python.org/en/latest/discussions/versioning/).
```

PEPs are historical documents, packaging.python.org should be used when possible. (Which is unfortunately not always, as some PEPs haven't been ported to there, I tried to fix that but there was not enough bandwidth to review the PRs…)

Reply:
The link you suggest is the first I found when searching information about python versioning. Eventually I decided to link to PEP 440, because it's what Briefcase recommends when building if you choose an unsupported version format. Ok, I'll change the link.